sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev_animators_100.json

```diff
@@ -0,0 +1 @@
+figure_hashes_mpl_dev_ft_261_astropy_dev_animators_dev.json
```

Comment:
This is due to the master version of mpl_animators returns 1.0.0 as a version for now.